src/workerd/api/tests/request-client-disconnect.js

```diff
@@ -0,0 +1,128 @@
+import { DurableObject, WorkerEntrypoint } from 'cloudflare:workers';
+import assert from 'node:assert';
+import { scheduler } from 'node:timers/promises';
```

Comment:
FWIW, `scheduler.wait` is actually available without node compat.

Reply:
VSCode just helpfully keeps importing it. I'll get rid of it before I merge.